### PR TITLE
Add option to enable timestamps for rich metadata info on discord

### PR DESCRIPTION
**Pull request by samwiseg0**
Added the ability to send timestamps to discord.

### CODE REVIEW

**Comment by JonnyWong16** [comment]
I don't think we need an option to include the timestamp. Just having it there all the time should be fine.

**Comment by samwiseg0** [comment]
ok i can remove the option then.

File: plexpy/notifiers.py [hunk 1]
```diff
@@ -1148,6 +1148,8 @@ def agent_notify(self, subject='', body='', action='', **kwargs):
             attachment = {'title': title
                           }
```
Comment (JonnyWong16) [inline plexpy/notifiers.py]: Just put the timestamp right inside the `attachment`.

Comment (samwiseg0) [reply plexpy/notifiers.py]: ok

**Comment by JonnyWong16** [comment]
Squash and it's good.

**Comment by samwiseg0** [comment]
I think it should be good now?

**Comment by JonnyWong16** [comment]
Um... I want one single commit.
https://gist.github.com/patik/b8a9dc5cd356f9f6f980#combining-the-commits

**Pull request closed without merging**